Src/Newtonsoft.Json.UnityConverters.Tests/ConvertingUnityTypes/VectorTests.cs

```diff
@@ -0,0 +1,59 @@
+﻿using UnityEngine;
+
+namespace Newtonsoft.Json.UnityConverters.Tests.ConvertingUnityTypes
+{
+    public class Vector2Tests : TypeTester<Vector2>
+    {
+        public static (Vector2 deserialized, string serialized)[] representations = new[] {
+            (new Vector2(1, 2), @"{""x"":1.0,""y"":2.0}")
+        };
+
+        protected override void ConfigureSettings(JsonSerializerSettings settings)
+        {
+        }
+    }
+
+    public class Vector2IntTests : TypeTester<Vector2Int>
+    {
+        public static (Vector2Int deserialized, string serialized)[] representations = new[] {
+            (new Vector2Int(1, 2), @"{""x"":1,""y"":2}")
+        };
+
+        protected override void ConfigureSettings(JsonSerializerSettings settings)
+        {
+        }
+    }
+
+    public class Vector3Tests : TypeTester<Vector3>
+    {
+        public static (Vector3 deserialized, string serialized)[] representations = new[] {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Change the visibility of 'representations' or make it 'const' or 'readonly'.](https://app.codacy.com/manual/jilleJr/Newtonsoft.Json-for-Unity.Converters/pullRequest?prid=4930242)